cogs/nerd.py

```diff
@@ -0,0 +1,33 @@
+import discord
+
+from discord.ext import commands
+
+from classes.discordbot import DiscordBot
+
+class Nerd(commands.Cog, name="nerd"):
+    """
+        He do be deserving it tho.
+
+        Require intents: 
+			- message_content
+		
+		Require bot permission:
+			- read_messages
+    """
+
+    def __init__(self, bot: DiscordBot) -> None:
+        self.bot = bot
+
+        self.target = "219156294974570497"
+        self.EMOJI = '🤓'
```

Comment:
You should be able to have multiple IDs and have them as well as the emoji be configured in the config files. This way you may even have a different emoji for each person.

Reply:
Invalid, it should only target you